networking/firewall/000infralib/src/lib/key.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+###########################################################
+# sshkey installing
+###########################################################
+network_set_sshkey()
```

Comment:
@idorax  do we even need network_set_sshkey if running on a single host, this test is not enabled in a multi-host environment, can you check with the networking team to see if we can remove anything related to installing ssh keys ? I know I did this with another test as it wasn't needed.

Reply:
Agreed with @rasibley , this whole file doesn't look needed for single host

Reply:
Hi @rasibley, file key.sh is removed.